Modules/Loadable/Markups/MRML/vtkMRMLMarkupsDisplayNode.cxx

```diff
@@ -231,6 +243,10 @@ void vtkMRMLMarkupsDisplayNode::CopyContent(vtkMRMLNode* anode, bool deepCopy/*=
   vtkMRMLCopyFloatMacro(LineColorFadingSaturation);
   vtkMRMLCopyFloatMacro(LineColorFadingHueOffset);
   vtkMRMLCopyBooleanMacro(HandlesInteractive);
+  vtkMRMLCopyBooleanMacro(FillVisibility);
+  vtkMRMLCopyBooleanMacro(OutlineVisibility);
+  vtkMRMLCopyFloatMacro(FillOpacity);
+  vtkMRMLCopyFloatMacro(OutlineOpacity);
```

Comment:
Could you add it to PrintSelf method, too?

Reply:
Done!